<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeckSet</class>
 <widget class="QMainWindow" name="DeckSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_p1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>PlayerName</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Billy.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Billy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Peketon.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Reketon</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Angus.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Angus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Duraham.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Duraham</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Billy.png);
image: url(:/new/prefix1/heros/Colonel Baba.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:
 Colonel Baba</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Medusa.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Medusa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Bunka.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Bunka</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sanka.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Sanka</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sir Lamorak.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Sir Lamorak</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Kabu.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Kabu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Salih.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Salih</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Rajakal.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Rajakal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Khan.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Khan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Boi.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Boi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Eloi.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Eloi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Kanar.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Kanar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Elsa.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Karissa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Karissa.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Elsa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sir Philip.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Sir Philip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Frost.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Frost</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Tusk.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Tusk</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Rambo.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Flying:Rambo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Death.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Floating:Death</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sabrina.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Floating:Sabrina</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set your deck</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="playerSwitch_1_pushBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1's deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play_pushBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>720</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playerSwitch_2_pushBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2's deck</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_p2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>PlayerName</string>
    </property>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Billy.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Billy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Peketon.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Reketon</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Angus.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Angus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Duraham.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Duraham</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Billy.png);
image: url(:/new/prefix1/heros/Colonel Baba.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:
 Colonel Baba</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Medusa.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Medusa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Bunka.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Bunka</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>410</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sanka.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Water Walking:Sanka</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sir Lamorak.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Sir Lamorak</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Kabu.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Kabu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Salih.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Salih</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Rajakal.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Rajakal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Khan.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Khan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Boi.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Boi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Eloi.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Eloi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>170</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Kanar.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Kanar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Elsa.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Karissa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Karissa.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Elsa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sir Philip.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Sir Philip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_46">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Frost.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Frost</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>270</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Tusk.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Grounded:Tusk</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Rambo.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Flying:Rambo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_49">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Death.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Floating:Death</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>75</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/heros/Sabrina.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Floating:Sabrina</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
